packages/prescriptionListLambda/tests/test_patientDetailsLookupService.test.ts

```diff
@@ -0,0 +1,612 @@
+import {
+  jest,
+  describe,
+  it,
+  expect,
+  beforeEach
+} from "@jest/globals"
+import {getPdsPatientDetails} from "../src/services/patientDetailsLookupService"
+import {Logger} from "@aws-lambda-powertools/logger"
+import {AxiosInstance, AxiosResponse} from "axios"
+import {PDSError} from "../src/utils/errors"
+
+// Create mock axios instance with get method
+const mockGet = jest.fn(() => Promise.resolve({} as unknown as AxiosResponse))
+const mockAxiosInstance = {
+  get: mockGet
+}
+
+// Mock the axios module
+jest.mock("axios", () => ({
+  create: () => mockAxiosInstance
+}))
+
+describe("Patient Details Lookup Service Tests", () => {
+  const mockLogger = new Logger({serviceName: "test"})
+  const mockEndpoint = "http://test-endpoint/personal-demographics/FHIR/R4"
+  const mockAccessToken = "test-token"
+  const mockRoleId = "test-role"
+  const mockNhsNumber = "9000000009"
+
+  const mockPdsBundle = {
```

Comment:
You're better off pulling this out into a `mockObjects` file, rather than defining it in the tests. It'll probably be easier to pull the mock endpoint, token, roleID and nhs number to the same file as well, and import them all together.